marconi-mamba/examples/json-rpc-server/src/Main.hs

```diff
@@ -0,0 +1,29 @@
+{-
+-- Sample JSON-RPC server program
+--  uncomment TODO and provide adequte data
+-}
+module Main where
+
+import Marconi.Api.Types (TargetAddresses)
+import Marconi.Bootstrap (bootstrapHttp, bootstrapJsonRpc, targetAddressParser)
+
+{-
+-- white space separated list of addresses
+-}
+
+-- TODO
+bech32Addresses :: String   -- ^  valid address to keep track of
+bech32Addresses = "bc1qrp33g0q5c5txsp9arysrx4k6zdkfs4nce4xj0gdcccefvpysxf3qccfmv3"
+
+-- TODO
+dbpath :: FilePath -- ^ valid SQLite marconi UTxo database path
+dbpath = "/home/kayvan/tmp/.db/main_utxoDB"
```

Comment:
Can you please change this to something else? Preferably something in the source tree.